emacs/cquery.el

```diff
@@ -371,10 +380,21 @@
         (buffer-string))
     (error str)))
 
+(defun cquery--render-objc-string (str)
+  (condition-case nil
+      (with-temp-buffer
+        (delay-mode-hooks (objc-mode))
+        (insert str)
+        (font-lock-ensure)
+        (buffer-string))
+    (error str)))
+
 (defun cquery--initialize-client (client)
   (dolist (p cquery--handlers)
     (lsp-client-on-notification client (car p) (cdr p)))
-  (lsp-provide-marked-string-renderer client "c++" #'cquery--render-string))
+  (lsp-provide-marked-string-renderer client "c" #'cquery--render-c-string)
```

Comment:
Why not just reuse the same cquery--render-string function three times?

Reply:
because of the language - ill make a generator for it though